Sources/SwiftCliCore/StitchLibrary.swift

```diff
@@ -1,6 +1,22 @@
 
 import Foundation
 
-let allowed_stitches = ["k1", "p1", "\n"]
+let allowed_stitches = ["k1", "p1",  "ssk", "k2tog", "yo", "m1", "\n"]
 let stitchPrinting = ["k1":" ",
-                      "p1":"-"]
+                      "p1":"-",
+                      "ssk":"\\",
+                      "k2tog":"/",
+                      "yo":"o",
+                      "m1":"m",]
+
+
+public func arrayCompare(array1: [[String]], array2: [[String]]) -> Bool {
+    var IsItTheSame : Bool! = nil
+    if (array1 == array2){
+        IsItTheSame = true
+    }
+    else{
+        IsItTheSame = false
+    }
+    return IsItTheSame
```

Comment:
You can just `return array1 == array2`.

Reply:
I realized I don't need this function either, I created it based out of an earlier mistaken assumption I couldn't compare two arrays in test cases. I don't remember why I thought that, seems silly now. I'm going to get rid of it.